adapters/mqtt-vertx/src/test/java/org/eclipse/hono/adapter/mqtt/impl/VertxBasedMqttProtocolAdapterTest.java

```diff
@@ -257,37 +306,60 @@ public void testMapTopicSupportsShortAndLongTopicNames(final VertxTestContext ct
 
     }
 
-    private void givenAnAdapter() {
+    /**
+     * Verifies that the adapter calls the mapper if configured. Also the updated payload and deviceId is checked.
+     *
+     * @param ctx The helper to use for running tests on vert.x.
+     */
+    @Test
+    public void testMapperShouldUpdatePayloadAndDeviceId(final VertxTestContext ctx) {
```

Comment:
this doesn't seem to be testing any functionality of the MQTT adapter, i.e. this should be a test case of `HttpBasedMessageMappingTest`, right?